src/style/index.css

```diff
@@ -0,0 +1,11 @@
+/* abstracts */
+@import url(./abstracts/variables.css);
```

Comment:
Please create index.css file in abstracts folder and then import variables.css file into it